recipes/level-zero/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This file is not needed. conda-build can find the one from the source tarball.

Reply:
Thanks, removed